apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -313,6 +313,14 @@ type DatacenterOptions struct {
 	// a user-provided monitoring solution (at present, only support for Prometheus is available).
 	// +optional
 	Telemetry *telemetryapi.TelemetrySpec `json:"telemetry,omitempty"`
+
+	// AdditionalContainers defines additional containers to be deployed in each Cassandra pod.
+	// +optional
+	AdditionalContainers []corev1.Container `json:"additionalContainers,omitempty"`
+
+	// AdditionalInitContainers defines additional init containers to be deployed in each Cassandra pod.
+	// +optional
+	AdditionalInitContainers []corev1.Container `json:"additionalInitContainers,omitempty"`
```

Comment:
To be consistent with volumes, and since we're now allowing to specify standard k8ssandra containers (to control the ordering or add settings), should we rename these to `Containers` and `InitContainers` instead?
The presence of `Additional` feels misleading now.

Reply:
Sounds good to me